.github/dependabot.yml

```diff
@@ -14,56 +14,8 @@ updates:
     directory: "/"
     schedule:
       interval: weekly
-    target-branch: "v0.37.x"
+    target-branch: "v0.34.x-celestia"
```

Comment:
maybe also this can be removed since we no longer want to maintain 0.34.x. @evan-forbes to confirm

Reply:
We need to maintain v0.34.x-celestia indefinitely because it is used by celestia-app v3.x and we use celestia-app v3.x to sync from genesis.